react/src/components/toggle/SprkToggle.js

```diff
@@ -105,7 +106,9 @@ SprkToggle.propTypes = {
    * Assigned to the `data-analytics` attribute serving as a unique selector for outside libraries to capture data.
   */
   analyticsString: PropTypes.string,
-  /**
+  /** Whether if the toggle is open or not */
```

Comment:
Let's  clarify this description by saying something like,

"Determines if the toggle is open upon loading on the page."